optimize/self-managed/optimize-deployment/configuration/system-configuration-platform-7.md

```diff
@@ -64,8 +64,8 @@ REST API endpoint locations, timeouts, etc.
 | import.skipDataAfterNestedDocLimitReached                 | false         | Some data can no longer be imported to a given document if its number of nested documents has reached the configured limit. Enable this setting to skip this data during import if the nested document limit has been reached.                                                                                                                                                                                                                                                                                            |
-| import.elasticsearchJobExecutorThreadCount                | 1             | Number of threads being used to process the import jobs per data type that are writing data to elasticsearch.                                                                                                                                                                                                                                                                                                                                                                                                             |
-| import.elasticsearchJobExecutorQueueSize                  | 5             | Adjust the queue size of the import jobs per data type that store data to elasticsearch. If the value is too large it might cause memory problems.                                                                                                                                                                                                                                                                                                                                                                        |
+| import.elasticsearchJobExecutorThreadCount\*              | 1             | Number of threads being used to process the import jobs per data type that are writing data to elasticsearch.                                                                                                                                                                                                                                                                                                                                                                                                             |
+| import.elasticsearchJobExecutorQueueSize\*                | 5             | Adjust the queue size of the import jobs per data type that store data to elasticsearch. If the value is too large it might cause memory problems.                                                                                                                                                                                                                                                                                                                                                                        |
```

Comment:
Lets also adjut the description of these (and any other shared DB settings) to say `database` or `elaticsearch/opensearch`